src/util/read-package-json.js

```diff
@@ -0,0 +1,7 @@
+import fs from 'fs-promise';
+import path from 'path';
+
+export default (dir) => {
```

Comment:
I think in order to make this function async, we'd have to add something like the `json-promise` package.